data/language/dutch.txt

```diff
@@ -1078,20 +1078,20 @@ STR_1076    :Winkelstand
 STR_1077    :Draaistand
 STR_1078    :Voorwaartse rotatie
 STR_1079    :Achterwaartse rotatie
-STR_1080    :Film: {ENDQUOTES}Avenging aviators{ENDQUOTES}
-STR_1081    :3D-film: {ENDQUOTES}Mouse tails{ENDQUOTES}
+STR_1080    :Film: {ENDQUOTES}Avenging Aviators{ENDQUOTES}
+STR_1081    :3D-film: {ENDQUOTES}Mouse Tails{ENDQUOTES}
 STR_1082    :Ruimteringenmodus
 STR_1083    :Beginnersstand
-STR_1084    :LIM-lancering
-STR_1085    :Film: {ENDQUOTES}Thrill riders{ENDQUOTES}
-STR_1086    :3D-film: {ENDQUOTES}Storm chasers{ENDQUOTES}
-STR_1087    :3D-film: {ENDQUOTES}Space raiders{ENDQUOTES}
+STR_1084    :Lineaire inductiemotor-lancering
```

Comment:
Ik was deze vergeten nog aan te passen, maar zoals het er nu staat is het gewoon correct. 'Lineaireinductiemotorlancering' is trouwens pertinent incorrect. Zie deze zin die je naar ik aanneem zelf geplaatst hebt? 
STR_0554 :Het karretje wordt het station uitgelanceerd over een lange rechte baan door middel van lineaire inductiemotoren, om daarna recht omhoog te gaan en weer terug te vallen richting het station

Maar als we 'LIM-lancering' laten staan hoeven we deze discussie niet eens te voeren. Inductie is volgens mij trouwens tegenwoordig wel zeer bekent dankzij fornuizen met inductie, i.t.t. 15 jaar geleden. Dus daarom wel verduidelijkend denk ik. Al gaat het inderdaad vooral om de lancering.